COMMENTS.md

```diff
@@ -0,0 +1,4 @@
+- [ ] Ensure to remove hover style for style.jsx and PlayList.jsx
```

Comment:
This is nice to have but can we have this as `TODO.md` instead?

Or best still, some time later I would add a more comfortable `changelog`

Reply:
BTW, excellent addition 💪
Let's have it user-scoped... This way I can add mine as well